test/model/autocomplete_test.dart

```diff
@@ -438,7 +438,7 @@ void main() {
     });
   });
 
-  group('MentionAutocompleteView sorting users results', () {
+  group('MentionAutocompleteView sorting user mention results', () {
```

Comment:
```suggestion
  group('MentionAutocompleteView sorting mention results', () {
```

(since wildcard mentions aren't user mentions)